client/src/main/java/io/pravega/client/stream/ReaderGroup.java

```diff
@@ -177,6 +177,15 @@
      */
     ReaderSegmentDistribution getReaderSegmentDistribution();
 
+    /**
+     * Publishes the given stream-cut to the controller in order to truncate the given stream if
```

Comment:
Maybe in this javadoc we dont need to tell the user where the stream cut is published (controller). We can simply say users can use this api to indicate to pravega that they have consumed data upto a specific streamcut and it is safe for pravega to truncate the stream from this readergroup's perspective.

Reply:
I've changed the javadoc to reflect this. Please have a look.